include/picongpu/fields/FieldJ.tpp

```diff
@@ -250,8 +244,8 @@ namespace picongpu
         deposit.template execute<T_area, numWorkers>(cellDescription, depositionKernel, solver, jBox, pBox);
     }
 
-    template<uint32_t T_area, class T_CurrentInterpolation>
-    void FieldJ::addCurrentToEMF(T_CurrentInterpolation& myCurrentInterpolation)
+    template<uint32_t T_area, class T_CurrentInterpolationFunctor>
+    void FieldJ::addCurrentToEMF(T_CurrentInterpolationFunctor myCurrentInterpolationFunctor)
```

Comment:
Why did you remove the by reference `&`?

Reply:
It was a bit unnatural, but not incorrect originally. This parameter is a functor that's not only not modified internally, but is stateless generally. And with the changed version the call changed and it didn't compile with this reference to non-const. So changed to passing by value, whicj was imo better for the original version as well.

Reply:
Thanks for the explanation.